src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -56,8 +56,7 @@
 
 /* Client channel implementation */
 
-grpc_tracer_flag grpc_client_channel_trace =
-    GRPC_TRACER_INITIALIZER(false, "client_channel");
+grpc_core::TraceFlag grpc_client_channel_trace(false, "client_channel");
```

Comment:
I just realized that this approach violates the style guide:

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Noah, can you please send a follow-up PR to fix this?  Thanks!

Reply:
What do you propose for fixing this? To me this seems like a case where a small violation of the style guide is worth it to avoid unneeded complexity. 

My idea for a fix would be to create a separate header file with all constexpr tracer definitions, and then link to that from all places where a flag is needed.